packages/wdio-cli/src/commands/config.ts

```diff
@@ -105,9 +105,12 @@ const runConfig = async function (useYarn: boolean, yes: boolean, exit = false)
      * ensure wdio packages have the same dist tag as cli
      */
     if (pkg._requested && pkg._requested.fetchSpec) {
-        packagesToInstall = packagesToInstall.map((p) => p.startsWith('@wdio') || ['devtools', 'webdriver', 'webdriverio'].includes(p)
-            ? `${p}@${pkg._requested.fetchSpec}`
-            : p
+        const { fetchSpec } = pkg._requested
+        packagesToInstall = packagesToInstall.map((p) =>
+            (p.startsWith('@wdio') || ['devtools', 'webdriver', 'webdriverio'].includes(p)) &&
+            (fetchSpec.includes('beta') || fetchSpec.includes('next'))
```

Comment:
This would limit us in only using `beta` or `next` tags. It would be great if every tag can work. Therefor I suggest to try to match against versions, e.g. `/(v)?\d+\.\d+\.\d+/` something like that?